compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Util.hs

```diff
@@ -242,6 +242,9 @@ mkTFuns ts t = foldr (:->) t ts
 mkTApps :: Type -> [Type] -> Type
 mkTApps = curry (review _TApps)
 
+splitTApps :: Type -> (Type, [Type])
+splitTApps = view _TApps
```

Comment:
I added this because my ultimate goal is to remove the lensy stuff at some point.

Reply:
This isn't being used in this PR, is it?